web/html/src/manager/virtualization/guests/console/guests-console.js

```diff
@@ -53,6 +62,10 @@ class GuestsConsole extends React.Component<Props, State> {
     this.pageUnloading = false;
     this.openVirtSocket();
 
+    // Schedule token refresh a minute before its expiration
+    const tokenTime = getTokenLifetime(props.token);
+    this.intervalId = window.setInterval(this.refreshToken, tokenTime - 60000);
```

Comment:
Nitpick: what happens, when the token lifetime is set to less than 60 seconds?  :trollface:  Could this happen and who controls this?

Reply:
We set it to 5 minutes in the `WebSockifyTokenBuilder` java class: so we are totally in control of that value!